### ISSUE TITLE
Issue with upstream Helm Chart

### ISSUE THREAD

**Issue by bvandewalle** [grafana/loki#420]
**Describe the bug**
Used the Helm chart in this repository using standard helm 2.13.

```
➜  helm install --name loki infra/loki
```

After deployment, the promtail containers all fail with the following logs:

```
➜  kubectl logs loki-promtail-46w6v
level=error ts=2019-03-26T01:02:55.11721353Z caller=main.go:35 msg="error loading config" filename=/etc/promtail/promtail.yaml err="yaml: unmarshal errors:\n  line 1: field backoff_config not found in type config.Config"
```

**Comment by daixiang0** [comment]
sorry that #415 introduce it, now it fix.

**Comment by daixiang0** [comment]
@bvandewalle btw, where do you  see `infra/loki`?

**Comment by davkal** [comment]
Fixed via #418

**Comment by bvandewalle** [comment]
@daixiang0 This is where I copied my chart locally (`infra/loki`)
